module/Decision/view/decision/member/index.phtml

```diff
@@ -87,8 +87,22 @@ case Member::TYPE_HONORARY:
     </table>
     <h3><?= $this->translate('Membership of committees and fraternities') ?></h3>
 <ul>
-<?php foreach ($member->getOrganInstallations() as $install): ?>
-    <li><?= $install->getFoundation()->getAbbr() ?></li>
+<?php
+$memberships = array();
+foreach ($member->getOrganInstallations() as $install) {
+    if (null !== $install->getDischargeDate()) {
+        continue;
+    }
+    if (!isset($memberships[$install->getOrgan()->getAbbr()])) {
+        $memberships[$install->getOrgan()->getAbbr()] = array();
+    }
+    if ($install->getFunction() != 'Lid') {
+        $memberships[$install->getOrgan()->getAbbr()][] = $install->getFunction();
+    }
+}
+?>
+<?php foreach ($memberships as $organ => $functions): ?>
```

Comment:
Might be better to just do this in the controller and then just return the $memberships variable to the view.